dotCMS/src/test/java/v2/languages/LanguagesResourceTest.java

```diff
@@ -38,8 +40,12 @@ public void list() throws DotSecurityException, DotDataException {
         when(initDataObject.getUser()).thenReturn(user);
         when(webResource.init(request, httpServletResponse,true)).thenReturn(initDataObject);
         when(languageAPI.getLanguages()).thenReturn(languages);
+        final LanguageView languageView = mock(LanguageView.class);
 
-        final LanguagesResource languagesResource = new LanguagesResource(languageAPI, webResource);
+        final LanguagesResource languagesResource = spy(new LanguagesResource(languageAPI, webResource));
+        Mockito.doReturn((Function<Language, LanguageView>) language -> languageView)
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'Mockito.doReturn' due to existing static import 'org.mockito.Mockito.*'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5971966)